tests/cases/fourslash/goToDefinitionJsxNotSet.ts

```diff
@@ -0,0 +1,17 @@
+/// <reference path='fourslash.ts' />
+
+// Regresion tests for GH#46854
+
+// @allowJs: true
+
+// @Filename: /foo.jsx
+//// const /*def*/Foo = () => (
+////     <div>foo</div>
+//// );
+//// export default Foo;
+
+// @Filename: /bar.jsx
+//// import Foo from './foo';
+//// const a = <[|/*use*/Foo|] />
+
+verify.goToDefinition("use", "def");
```

Comment:
Is this test enough to reproduce the issue, even though there's no jsconfig in this test? From the issue, I thought the bug only happened if we had a jsconfig.